tests/rest/RSTcheckBearTest.py

```diff
@@ -7,6 +10,22 @@
 python_block_good = '====\nTest\n====\n.. code-block:: python\n\n    print()'
 python_block_bad = '====\nTest\n====\n.. code-block:: python\n\n    print(\n'
 
+
+good_file2 = ':doc:`here <../Users/Tutorial>`'
+bad_file2 = '==test\n==='
+
+
+def test_ignore_unknown(errors):
+    for error in errors:
+        msg = error.full_message
+    unittest.assertRegex('[\w|\-]+', re)
+    unittest.assertEqual(
+        msg,
+        'No directive entry for "argparse" in module'
+        '"docutils.parsers.rst.languages.en"'
+    )
```

Comment:
tests need to be defined inside a test class ;) That's why you don't get 100% coverage, as this function never gets picked up. But do you need `_ignore_unknown` anyway? It's never used